components/renku_data_services/users/core.py

```diff
@@ -9,16 +9,18 @@ def validate_unsaved_secret(body: apispec.SecretPost) -> UnsavedSecret:
     secret_kind = SecretKind(body.kind.value)
     return UnsavedSecret(
         name=body.name,
-        default_filename=body.default_filename,
         secret_value=body.value,
         kind=secret_kind,
+        expiration_timestamp=body.expiration_timestamp,
+        default_filename=body.default_filename,
     )
```

Comment:
issue: We should not accept an expiration time without timezone information. Then we do not know when the credentials truly expire. The code generated from the apispec cannot validate this I think.